app/assets/javascripts/application.js

```diff
@@ -1,107 +1,121 @@
 //= require jquery
 //= require textinputs_jquery
 //= require jquery-cookie/jquery.cookie
+//= require three.min
+//= require fort-scene
 
-function anim_show(el) {
-  $(el).fadeIn();
-}
+(function() {
+  function anim_show(el) {
```

Comment:
Identifier 'anim_show' is not in camel case.<br>'anim_show' is defined but never used.